test/test/comptask/ComptaskTests.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.comptask;
+
+import one.profiler.test.*;
+
+public class ComptaskTests {
+    @Test(
+        mainClass = Main.class,
+        agentArgs = "start,features=comptask,event=Compile::Init,collapsed",
```

Comment:
Let's use `event=cpu` here to focus on `comptask` rather than `Compile:Init` event.

Reply:
The reason why I've done it this way is that this will record a profile when `Compile::Init` is called, which happens whenever a method is JIT compiled. Using `event=cpu` means that we depend on the sample frequency, and it could happen that we miss the small time window in which the compiler threads are touching `Main#main` (quite often)

Reply:
I second @krk on profiling `cpu`, since this is how the feature is going to be used in practice. Profiling `cpu` will certainly add nondeterminism, but that's actually a good thing - in this way, we can verify that async-profiler is capable of injecting compilation task frame at different execution states and that it does not cause stability issues in various transient frame states (there were such issues in the past). Furthermore, this will make possible to test the feature on macOS.

Reply:
ce9b53bd955ca0382cd5294448ada67c9e5a9280